lib/axios.ts

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
+import { createSecretToken } from './cryptoUtil'
+
+export const axiosInstance = axios.create({
+  headers: {
+    'Content-Type': 'application/json',
+  },
+})
+
+axiosInstance.interceptors.request.use(
```

Comment:
Make a check here so that we only add secrectToken to the headers of the get request (our get request is only on server, so it will be able to access the server ENVIRONMENT VARIABLE)